fragments/filter/README.md

```diff
@@ -768,13 +756,13 @@ This uses the same queryables as Example 4.
 
 This uses the same queryables as Example 4.
 
-The only temporal operator required is `ANYINTERACTS`, which follows the same semantics as the existing 
-`datetime` parameter. This is effectively that the datetime or interval operands have any overlap between them.
+The only temporal operator required is `ANYINTERACTS`. This is effectively that the datetime or interval operands 
+have any overlap between them.
 
 #### Example 6: ANYINTERACTS cql-text (GET)
 
 ```javascript
-/search?filter=datetime ANYINTERACTS 2020-11-11
+/search?filter=datetime ANYINTERACTS 2020-11-11T00:00:00Z/2020-11-12T00:00:00Z
```

Comment:
these examples were just wrong

Reply:
What does this mean? They still need to be changed here?

Reply:
I fixed them. I was stating that the reason I fixed them was that they were just invalid expressions(e.g., `datetime ANYINTERACTS 2020-11-11` isn't valid because it's just a date and not a datetime range), rather than there was actually a change to the syntax.